scipy/sparse/_bsr.py

```diff
@@ -61,7 +61,6 @@ class _bsr_base(_cs_matrix, _minmax_mixin):
     indptr
         BSR format index pointer array
     blocksize
-        Block size
```

Comment:
Do these docstrings conflict with the property docstring? It seems strange to list the properties in this "Attributes" list but explicitly not describe what they are.

I'd prefer to either (a) not include properties here, or (b) include them but provide a short description.

Maybe neither of these will produce correctly rendered docs, so please let me know if this would be a problem.

Reply:
They sorta conflict with the property docstring. We need to have an entry for the attribute for it to show up, but the docs come from the property docstring. 

Here's what the rendered attribute docs look like on the current commit:

<img width="910" alt="image" src="https://github.com/scipy/scipy/assets/8238804/8ba95125-a1c1-4709-a72b-0478efdd0374">

Reply:
I guess my concern is that some users won't look at the rendered docs, and will only see the docstrings in the source itself. (I'm definitely more likely to look at source than rendered docs myself.)

Maybe we can tolerate a bit of duplication and describe each attribute (briefly!) in the docstring even though that part won't appear in the rendered docs.

Reply:
On further consideration, I don't think this is a big deal. Resolving.